store/src/java-test/com/zimbra/cs/imap/ImapHandlerTest.java

```diff
@@ -105,10 +115,18 @@ public void testDoCOPYByUID() throws Exception {
        Assert.assertEquals("original messages should have stayed in inbox", Mailbox.ID_FOLDER_INBOX, mbox.getMessageById(null, m1.getId()).getFolderId());
        Assert.assertEquals("original messages should have stayed in inbox", Mailbox.ID_FOLDER_INBOX, mbox.getMessageById(null, m2.getId()).getFolderId());
        Assert.assertEquals("original messages should have stayed in inbox", Mailbox.ID_FOLDER_INBOX, mbox.getMessageById(null, m3.getId()).getFolderId());
+        } catch (Exception e) {
+            fail("No error should be thrown");
+            e.printStackTrace();
+        }
     }
 
     @Test
     public void testDoCOPYByNumber() throws Exception {
+        
+       Account acct = Provisioning.getInstance().getAccount("12aa345b-2b47-44e6-8cb8-7fdfa18c1a9f");
```

Comment:
Usually we get the Account object by the email, something like 
```
Account acct = Provisioning.getInstance().get(Key.AccountBy.name, LOCAL_USER);
/*  LOCAL_User = "localimaptest@zimbra.com" */
```
Perhaps "get by name" is better visibility than "get by account id).